Tutorial.md

```diff
@@ -11,477 +12,247 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
+                        "url": "/questions/1/choices/1",
+                        "votes": 2048
+                    }, {
+                        "choice": "Python",
+                        "url": "/questions/1/choices/2",
+                        "votes": 1024
+                    }, {
+                        "choice": "Objective-C",
+                        "url": "/questions/1/choices/3",
+                        "votes": 512
+                    }, {
+                        "choice": "Ruby",
+                        "url": "/questions/1/choices/4",
+                        "votes": 256
+                    }
+                ]
             }
+        ]
+```
 
-    + Response 200
+The polls resource has a second action, allowing you to create a new question. Inside this action, there is a description for the structure you would send to the server to create a new question.
 
-        [Gist][]
+```markdown
+### Create a new question [POST]
 
-    ### Delete a Gist [DELETE]
-    + Response 204
+You can create your own question using this action. It takes a JSON dictionary containing a question and a collection of answers in the form of choices.
 
-Here are a few new things to learn about payloads:
+- question (string) - The question
+- choices (array[string]) - A collection of choices.
+```
 
-#### Request and its Payload
+This action takes a JSON payload as part of the request as follows:
 
-    + Request (application/json)
+```markdown
++ Request (application/json)
 
             {
-                "content": "Updated file contents"
+                "question": "Favourite programming language?",
+                "choices": [
+                    "Swift",
+                    "Python",
+                    "Objective-C",
+                    "Ruby"
+                ]
             }
+```
 
-Our "Edit a Gist" action needs to receive some input data to update the state of the Gist resource with it. The data are part of an HTTP request message. This section defines such a Request and an example [payload][] of the request message.
-
-> **NOTE:** The Request follows exactly the same structure as a Response or Model. As with the Resource Model a Request may not include a response status code.
-
-#### Empty Payload
-
-    ### Delete a Gist [DELETE]
-    + Response 204
-
-In the case of the "Delete a Gist" action our response bears status code "204" indicating the server has successfully fulfilled the request and that there is no additional content to be send in the response payload body.
-
-## Collection Resource
-With single the Gist Resource in place let's define the collection of our Gist Resources:
-
-    ## Gists Collection [/gists{?since}]
-    Collection of all Gists.
-
-    The Gist Collection resource has the following attribute:
-
-    - total
-
-    In addition it **embeds** *Gist Resources* in the Gist Fox API.
-
-
-    + Model (application/hal+json)
+This example returns a 201 status code, along with HTTP headers and a JSON Body.
```

Comment:
Above you had `200` as inline code but here you have 201 as just text. I'd vote include code again. Additionally, no need to capitalize "Body."